feathr_project/registry/registry/models.py

```diff
@@ -0,0 +1,707 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Optional, Union
+from uuid import UUID
+import json
+import re
+
+
+def to_snake(d):
```

Comment:
Add description for this method. Also we should add hint for the input/output types.